src/data/types/drupal.d.ts

```diff
@@ -0,0 +1,418 @@
+import { DrupalFile, DrupalMedia, DrupalNode } from "next-drupal"
```

Comment:
This is a types file for Drupal-specific structures. These are intended to verify that the data we receive from Drupal contains the information we expect.

This might want to be broken up into multiple files. It's already hard to navigate, and we're going to have many more types here.